### DISCUSSION TITLE
Privileges problem between LCX's and a host

### DISCUSSION THREAD

**Discussion by Lum1r** [tteck/Proxmox#2419] [Help Needed]
Hello !

sorry if this post structure seems chaotic, i was doing my best to describe it as best as possible, so if it looks like lazyness in fact woul be my lack of profficiency in linux :) 

Im struggling with some issues on LCX containers created by scripts written by almighty ttek.
I would like to have multiple containers with read/write privileges on directory mouted on a host (4TB SATA HDD)
on every CT ive created mountpoint in these manner : mp0: /mnt/pve/hdd,mp=/mnt/hdd
Containers are able to see directories created by other CT or host on disk, but when checking the ownership of a directory created elswere than currently looked CT, it shows "nobody".
I sumbled across the article of mapping UIDs and GIDs:
https://pve.proxmox.com/wiki/Unprivileged_LXC_containers

but not sure how to implement that solution for it to work on my system. Im assuming every script (example deluge client) creates LCX where it runs as a service under root?

my question is, what by Your suggestion would be the easiest method of allowing multiple containers having w/r access to this specific drive, for me to be able to automate several services (deluge, sonar, radar, small fileserver) to have above access simultaneously?

If any additional data needed please let me know, i will do my best.
Cheers!

**Comment by Kelsch** [comment]
Were you able to figure this out? I got a system that works for me, but whenever I download something with deluge it is unaccessible to anything else.
